src/core/utils/qfieldcloudutils.h

```diff
@@ -16,10 +16,13 @@
 #ifndef QFIELDCLOUDUTILS_H
 #define QFIELDCLOUDUTILS_H
 
+#define ERROR_CODE_OVER_QUOTA QString( "over_quota" )
```

Comment:
```suggestion
#define ERROR_CODE_OVER_QUOTA QStringLiteral( "over_quota" )
```

But I thinki a `const static QString { QStringLiteral( "..." ) }` would be even better

Reply:
Done. :+1: